sqlobject/src/main/java/org/jdbi/v3/sqlobject/HandlerFactory.java

```diff
@@ -16,13 +16,15 @@
 import java.lang.reflect.Method;
 import java.util.Optional;
 
+import org.jdbi.v3.core.config.internal.Injected;
+
 /**
  * Creates Handler objects for methods annotated with a specific SQL method annotation, which satisfy the contract of
  * that annotation.
  *
  * @see SqlOperation
  */
-public interface HandlerFactory {
+public interface HandlerFactory extends Injected {
```

Comment:
HandlerFactory is stateless--you are adding config state to objects that get reused between many threads. This wants to be a parameter to `buildHandler`.